src/app/games/mancala/kalah/KalahTutorial.ts

```diff
@@ -0,0 +1,83 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { MancalaState } from '../commons/MancalaState';
+import { KalahMove } from './KalahMove';
+import { MancalaDistribution } from '../commons/MancalaMove';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+import { MancalaTutorial } from '../commons/MancalaTutorial';
+
+export class KalahTutorial extends Tutorial {
+
+    public gameName: string = $localize`Kalah`;
+
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            this.gameName,
+            $localize`${this.gameName} is a Mancala. Mancala is the name of a family of board games that date back at least to the third century. Mancalas are games of distribution (sowing) and capture. Their goal is to capture the most seeds. The spaces in Mancalas are called the houses. As you're playing Dark, the 6 houses on the bottom are yours.`,
```

Comment:
date -> dates
you're -> you are (you're c'est SMSant)

Reply:
137 you're
108 you are !
Et je dit ça après avoir modifié ceux là :p !